### DISCUSSION TITLE
Bug Report: Empty Post Submission Sends Last Post Content

### DISCUSSION THREAD

**Discussion by jatinkaushik-jk** [Avocodos/avocodos#11] [General]
### **Bug Report: Empty Post Submission Sends Last Post Content**

**Description:**
When attempting to post an empty message, instead of blocking the submission, the last successfully posted message is sent again as a new post.

**Steps to Reproduce:**
1. Go to the post creation section of the app.
2. Leave the post content input field blank.
3. Press the "Post" button.
4. The app submits the last successfully posted message, instead of showing an error or blocking the submission.

**Expected Behavior:**
When submitting an empty post, an error message should appear to inform the user that content is required, and the post should not be submitted.

**Actual Behavior:**
The last posted message is automatically sent as a new post when trying to submit an empty post.

**Video Proof:**

https://github.com/user-attachments/assets/02458376-8c6c-45ef-831d-8e8eac0b3461

**Severity:**
Moderate – This bug could lead to unintentional duplicate posts.

**Additional Notes:**
This issue might be caused by a caching mechanism or failure to clear the post data buffer after a successful submission.

**Comment by HarjjotSinghh** [comment]
Hmm, looks like I forgot to set the input back to an empty string after successful post submission 😅. Will fix it in the next commit in a few minutes!

**Comment by HarjjotSinghh** [comment]
This has now been fixed with the commit d18b22e. Thanks for finding and raising this issue!
